db/seeds/evaluatieonderzoek/questionnaires/evaluatieonderzoek.rb

```diff
@@ -1,85 +1,945 @@
-                         '25 West-Kennemerland',
-                         '26 Zuid-Holland-Noord',
-                         '27 Zuid-Holland-Oost',
-                         '28 Haaglanden',
-                         '29 Rijnmond',
-                         '30 Zuid-Holland-Zuid',
-                         '31 Oosterschelde regio',
-                         '32 Walcheren',
-                         '33 Zeeuwsch-Vlaanderen',
-                         '34 West-Brabant',
-                         '35 Midden-Brabant',
-                         '36 Noord-Oost-Brabant',
-                         '37 Zuidoost-Brabant',
-                         '38 Gewest Limburg-Noord',
-                         '39 Gewest Zuid-Limburg'
+                         { title: 'Ja', shows_questions: %i[v38] },
+                         { title: 'Nee' }
+                       ],
+                       show_otherwise: false
+                     }, {
+                       id: :v38,
+                       hidden: true,
+                       type: :textarea,
+                       title: 'Wat is er veranderd sinds de RMC contactgemeente hiervoor verantwoordelijk is?',
+                       placeholder: 'Vul hier uw antwoord in'
+                     }, {
+                       section_start: '<div class="divider"></div><p class="flow-text"><strong>Maatregel 4</strong>: De RMC contactgemeente heeft de verantwoordelijkheid voor een deel van het regionaal budget: de kassiersrol van het regionaal budget wordt nu gedeeld door de school én door de RMC contactgemeente.</p>',
+                       id: :v39,
+                       hidden: true,
+                       type: :range,
+                       title: 'Heeft u in de praktijk iets gemerkt van de verandering in de verantwoordelijkheid voor het regionaal budget?',
+                       labels: ['Helemaal niks gemerkt', 'Heel veel gemerkt']
+                     }, {
+                       id: :v40,
+                       hidden: true,
+                       type: :checkbox,
+                       show_otherwise: false,
+                       title: 'In hoeverre nemen de RMC contactgemeente en de contactschool samen hun verantwoordelijkheid voor de besteding van het regionale budget binnen uw regio?',
+                       options: [{ title: 'Ik weet het niet', hides_questions: %i[v40_1] }]
+                     }, {
+                       id: :v40_1,
+                       hidden: true,
+                       type: :range,
+                       title: '',
+                       labels: ['Helemaal niet', 'Helemaal wel']
+                     }, {
+                       id: :v41,
+                       hidden: true,
+                       type: :checkbox,
+                       show_otherwise: false,
+                       title: 'Hoe ervaart u de kwaliteit van de besteding van het regionale budget?',
+                       options: [{ title: 'Ik weet het niet', hides_questions: %i[v41_1] }]
```

Comment:
wat denk jij, willen we hier een soort extra generator voor maken die dit automatisch oplost? of het misschien inbakken in de range? Of het gewoon zo laten?

Reply:
Ik zou zeggen zo laten. Vanwege de complexiteit (en compatibiliteit met bijv de mobile app oid die ook ons vragenlijstending moeten parsen). Ook komt dit voor bij meerdere vraagtypen, soms is de vraag namelijk een range, soms is het een textarea (dat komt later voor). Dan zou je dus al die typen moeten aanpassen. Nu is het wel mooi transparent (maar wel error prone).

Ik heb de layout nu wel een beetje mooier gemaakt door de elementen dichter bij elkaar te zetten: als een `question` hash een `title` attribuut heeft, maar dat `title` attribuut is `blank?`, dan gooit ie er nu een `no-title` css klasse overheen wat het tweede element wat naar boven schuift op de pagina.